test/blackbox-tests/test-cases/pkg/opam-package-with-files-install.t

```diff
@@ -0,0 +1,22 @@
+We make sure that file like foo.install are correctly included from the files
+subdirectory of an opam repository. Notably, the build step here is empty. The
+library seq is such an example in the wild with this behaviour.
+
+  $ . ./helpers.sh
+  $ mkrepo
+
+  $ mkpkg foo
+  $ mkdir -p $mock_packages/foo/foo.0.0.1/files
+  $ cat > $mock_packages/foo/foo.0.0.1/files/foo.install
+
+  $ solve foo
+  Solution for dune.lock:
+  - foo.0.0.1
+
+The build action copying files/ is missing.
```

Comment:
There is no "build action" for copying files/

Reply:
You're right, I've rewritten it to something clearer now.